### PR TITLE
[encointer] set target blocktime to 6s

### PR DESCRIPTION
**Pull request by brenzi**
Encointer communities could benefit from 6s block time because the network is used for IRL point-of-sale or person-to-person transactions

Encointer is unaffected by https://github.com/paritytech/polkadot-sdk/issues/3268 as its pallets have since ever based time on block timestamps

The parameters are copy-paste from people, as introduced by #308

### CODE REVIEW

File: system-parachains/encointer/src/lib.rs [hunk 1]
```diff
@@ -752,6 +746,41 @@ construct_runtime! {
...
+
+/// We allow for 2 seconds of compute with a 6 second average block time.
+pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
+	WEIGHT_REF_TIME_PER_SECOND.saturating_mul(2),
+	polkadot_primitives::MAX_POV_SIZE as u64,
+);
+
+/// Maximum number of blocks simultaneously accepted by the Runtime, not yet included
+/// into the relay chain.
+pub const UNINCLUDED_SEGMENT_CAPACITY: u32 = 2;
```
Comment (clangenb) [inline system-parachains/encointer/src/lib.rs:778]: I have seen and implemented 3 for the unincluded segment capacity, but I remember learning that 3 and 2 actually resolve to the same value of 6s blocktime in the end.

**Review by eskimor — APPROVED** [review]
Looks good.

**Comment by brenzi** [comment]
CI fail out-of-scope: Check Migrations / check-migrations (asset-hub-kusama
but I can't trigger a re-run. Who could trigger that?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Allow signed origins to send arbitrary XCMs from some system chains ([polkadot-fellows/runtimes#407](https://github.com/polkadot-fellows/runtimes/pull/407))
 - Include the Core and Salary pallets into the Fellowship proxy ([polkadot-fellows/runtimes#454](https://github.com/polkadot-fellows/runtimes/pull/454))
 - Add new community democracy and treasuries pallets to Encointer ([polkadot-fellows/runtimes#456](https://github.com/polkadot-fellows/runtimes/pull/456))
+- Change target block time for Encointer to 6s
```
Comment (joepetrowski) [inline CHANGELOG.md:14]:
```suggestion
- Change target block time for Encointer to 6s ([polkadot-fellows/runtimes#462](https://github.com/polkadot-fellows/runtimes/pull/462))
```

**Comment by bkchr** [comment]
@brenzi all the pallets you are using, none of them has scheduled anything based on blocks? Maybe your democracy or similar? You will produce twice as many blocks, meaning that anything that is using blocks for scheduling will happen twice as fast.

**Review by joepetrowski — APPROVED** [review]

**Review by joepetrowski — CHANGES_REQUESTED** [review]
Just removing my review for Basti's comment due to auto-merge.

**Comment by brenzi** [comment]
@bkchr I'm pretty sure we're fine:
* encointer_scheduler: schedules cycles based on block timestamps
* encointer_democracy: evaluates proposal timings based on timestamps
* encointer_balances: demurrage is defined per block and will effectively be doubled. However, demurrage rates can be changed by Encointer council, so state migration could be skipped. As I observed the peoplechain for a while, I saw that the average block time was in between 6 and 12s. I think it is reasonable to apply 6s target now and then adjust demurrage based on actual block time statistics when all went well.

**Review by kianenigma — COMMENTED** [review]
Can you please check if the instructions in https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/guides/async_backing_guide/index.html are enough to create this PR?

**Comment by brenzi** [comment]
@kianenigma that document encompasses much more than this PR and Encointer benefitted from many other PR's in this repo that did all the prep work around async backing. In this PR I basically just needed to change constants - and there the linked docs are fine

**Comment by bkchr** [comment]
Ty @brenzi. If you are using timestamps, it should all be good :)

**Review by joepetrowski — APPROVED** [review]

**Pull request merged**